static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -69,7 +70,9 @@ export const getRandomIntInclusive = (
     return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
-export const shouldIncludeAppNexus = (): boolean => isInAuRegion();
+export const shouldIncludeAppNexus = (): boolean =>
+    isInAuRegion() ||
+    (config.get('switches.prebidAppnexusUkRow') && !isInUsRegion());
```

Comment:
@katebee Good shout on the logic; I think it's now required because of this switch, but without it it wasn't. What do you think?